content/articles/data-representation-using-brython/index.md

```diff
@@ -0,0 +1,353 @@
+
+##### Chart.py file
+Having created our `chart.py` file, We import this file into our boilerplate using the script tag and set the type attribute to `"text/python3"`. This file is imported into our HTML boilerplate using the script tag below:
+```html
+<!-- chart.py -->
+<script type="text/python3" src="chart.py"></script>
+```
+
+The chart.py file contains the following code:
+
+```python
+from browser import window
+
+b_highchart = window.Highcharts.Chart.new
+b_highchart(
+    {
+        "chart": {
+            "plotBackgroundColor": "#FCFFC5",
+            "plotBorderWidth": None,
+            "plotShadow": False,
+            "renderTo": "container",
+        },
+        "title": {"text": "Religious Distribution in Nigeria"},
+        "tooltip": {"pointFormat": "<b>{point.percentage:.1f}%</b>"},
+        "plotOptions": {
+            "pie": {
+                "allowPointSelect": True,
+                "cursor": "pointer",
+                "dataLabels": {
+                    "enabled": True,
+                    "format": "<b>{point.name}</b>: {point.percentage:.1f} %",
+                    "style": {"color": "black"},
+                },
+            }
+        },
+        "series": [
+            {
+                "type": "pie",
+                "data": [
+                    ["Indigenious Beliefs", 10.7],
+                    ["Other Christian", 35.3],
+                    {
+                        "name": "Roman Catholicism",
+                        "y": 10.0,
+                        "sliced": True,
+                        "selected": True,
+                    },
+                    ["Islam", 43.5],
+                    ["Others", 0.5],
+                ],
+            }
+        ],
+    }
+)
+```
+The code above plots the pie chart using the following methods:
+- `from browser import window`: imports the window module from, thus allowing Brython access to Javascript object.
+- `plotBackgroundColor`: It plots the background color for the chart area.
+- `plotBorderWidth`: It plots the width of the chart area in pixels.
+- `plotShadow`: This method takes in boolean values and adds a drop shadow to the area. 
+- `renderTo`: This method is like `document.getElementBy` in javascript. It will take the `id` of the `div` tag in our HTML code.
+- `tooltip`: This object contains the details in the small box when we hover on our chart.
+- `allowPointSelect`: If set as `true`, it allows you to toggle or select any of the slices in the pie chart
+- `Series`: this is a set of data. In this case, it is the religious distribution of people in Nigeria.
+
+When we execute this code, our pie chart is as shown below:
+
+![piechart1](engineering-education/data-representation-with-react-and-ant-design/piechart1.png) 
+
+![piechart2](engineering-education/data-representation-with-react-and-ant-design/piechart2.png)
+
```

Comment:
```suggestion

![piechart1](engineering-education/data-representation-using-brython/piechart1.png) 

![piechart2](engineering-education/data-representation-using-brython/piechart2.png)

```

Reply:
Why is there a need for using 2 piechart diagrams that show the same information? Please remove the redundant one. And rename the remaining one from `piechart1` to `piechart`. Also, do that the image name.